playbooks/bibdata.yml

```diff
@@ -0,0 +1,26 @@
+---
+# by default this playbook runs in the staging environment
+# to run in qa, pass '-e runtime_env=qa'
+# to run in production, pass '-e runtime_env=production'
+- Build bibdata infrastructure
```

Comment:
Needs `name:` to fix the syntax error issue I saw and commented on earlier

Reply:
Although once this change is in then the playbook fails on 
```
TASK [postgresql : PostgreSQL | ensure access to postgres server for client connections] *************************************************************
fatal: [bibdata-qa1.princeton.edu -> lib-postgres3.princeton.edu]: FAILED! => {"changed": false, "msg": "Destination /etc/postgresql/13/main/pg_hba.conf does not exist !", "rc": 257}
fatal: [bibdata-worker-qa1.princeton.edu -> lib-postgres3.princeton.edu]: FAILED! => {"changed": false, "msg": "Destination /etc/postgresql/13/main/pg_hba.conf does not exist !", "rc": 257}

PLAY RECAP *******************************************************************************************************************************************
bibdata-qa1.princeton.edu  : ok=49   changed=3    unreachable=0    failed=1    skipped=19   rescued=0    ignored=0   
bibdata-worker-qa1.princeton.edu : ok=49   changed=3    unreachable=0    failed=1    skipped=19   rescued=0    ignored=0   
```

Reply:
D'oh! I fixed the syntax thing by adding `name:`, thanks for catching that @carolyncole.

I'll do some digging on the postgresql issue.

Reply:
Okay, the postgresql failure is a legit error. On the `lib-postgres3` box the installed version of pgsql is 10:
```
postgres@lib-postgres3:/home/pulsys$ ls -la /etc/postgresql/
total 20
drwxr-xr-x   3 postgres postgres  4096 Mar 15  2018 .
drwxr-xr-x 116 root     root     12288 Mar  8 15:22 ..
drwxr-xr-x   3 postgres postgres  4096 Mar 15  2018 10
postgres@lib-postgres3:/home/pulsys$ ls -la /etc/postgresql/10/main/
total 68
drwxr-xr-x 3 postgres postgres  4096 Mar 24 11:16 .
drwxr-xr-x 3 postgres postgres  4096 Mar 15  2018 ..
drwxr-xr-x 2 postgres postgres  4096 Jul 15  2021 conf.d
-rw-r--r-- 1 postgres postgres   315 Mar 15  2018 environment
-rw-r--r-- 1 postgres postgres   143 Mar 15  2018 pg_ctl.conf
-rw-r----- 1 postgres postgres  9968 Mar 24 11:16 pg_hba.conf
-rw-r----- 1 postgres postgres  1636 Mar 15  2018 pg_ident.conf
-rw-r--r-- 1 postgres postgres 22538 Sep 28  2020 postgresql.conf
-rw-r--r-- 1 root     root       681 Mar 17  2018 repmgr.conf
-rw-r--r-- 1 postgres postgres   317 Mar 15  2018 start.conf
```
Should the bibdata boxes be looking at a different pgsql server?